math/dec.go

```diff
@@ -926,3 +926,25 @@ func FormatDec(v string) (string, error) {
 
 	return intPart + "." + decPart, nil
 }
+
+func ParseDec(v string) (string, error) {
+	parts := strings.Split(v, ".")
```

Comment:
Can you move parseDec to `tx/textual/valuerenderer/dec.go` too? I think with Aaron we discussed whether the `'` for thousands separator was a good idea, and let's make that decision later